whitepaper/interledger.aux

```diff
@@ -0,0 +1,46 @@
+\relax 
```

Comment:
When working with git + tex, I prefer to commit only source files like .tex and .bib. I put intermediary and compiled files like .dvi, .aux, .pdf etc. into .gitignore.